vendor/vendor.json

```diff
@@ -852,10 +852,10 @@
 			"revisionTime": "2016-12-14T09:25:55Z"
 		},
 		{
-			"checksumSHA1": "FEzQdhqmb6aqGL1lKnjOcUHIGSY=",
+			"checksumSHA1": "F/N7JzfuHZAk1UXi2zw/62hFPpM=",
 			"path": "google.golang.org/api/bigquery/v2",
```

Comment:
Seems like we're bringing in a lot of unnecessary vendor changes for this PR. Could we clean this up to just update storage, or is there a reason it needs to bring bigquery along?

Reply:
(Also, is a storage library update necessary to support this functionality?)

Reply:
Yes an update is necessary and this update bring bigquery lib with small change type